docs/data/data-grid/rows/rows.md

```diff
@@ -135,6 +135,26 @@ To add a border instead, set `rowSpacingType` to `"border"` and customize the co
 
 You can check the [styling rows](/x/react-data-grid/style/#styling-rows) section for more information.
 
+## Row reorder [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/)
+
+Row reorder is used to rearrange rows by dragging the row with the mouse.
+
+By default, `DataGridPro` does not allow row reordering. To enable reordering on all rows by dragging the special row reordering cells and moving it up or down, use the `rowReordering` prop.
```

Comment:
We need to consider `DataGridPremium` now, so better to avoid using the component name. For code examples, there's no escape.

```suggestion
By default, row reordering is disabled.
To enable it, you need to add the `rowReordering` prop.

<DataGridPro rowReordering />
```